dawn/docs/tutorial/README.md

```diff
@@ -0,0 +1,105 @@
+# Getting Started using GTClang & dawn
```

Comment:
My branch `https://github.com/VulcanClimateModeling/dawn/tree/jdahm-getting_started` has some corrections to this file.